code/modules/materials/definitions/solids/materials_solid_wood.dm

```diff
@@ -74,7 +75,9 @@
 	. += new/datum/stack_recipe/prosthetic/right_hand(src)
 	. += new/datum/stack_recipe/prosthetic/left_foot(src)
 	. += new/datum/stack_recipe/prosthetic/right_foot(src)
-	. += new/datum/stack_recipe/campfire(src)
+
+	if(stack_type == /obj/item/stack/material/log)
```

Comment:
`if(ispath(stack_type)) return` on lines 46-47 will make this not work. Maybe make it a separate override, or split that check up somehow? Remember we still need to check holographic and reinf_material.